docs/assets/extra.css

```diff
@@ -147,4 +147,17 @@ img.copyright-logo {
 
 .md-typeset .tabbed-set {
     border: 0.5px solid gray;
+}
+
+.md-typeset .tabbed-content {
+    padding-left: 0.5rem;
+    padding-right: 0.5rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+}
+
+.md-typeset .admonition,
+.md-typeset details {
+    border-width: 0;
+    border-left-width: 4px;
```

Comment:
Was the intent to remove the border on these elements so that it's only viewable on the left side?

![image](https://github.com/nautobot/pynautobot/assets/141754521/13b622e6-95e2-4648-a3d5-3c20e158ee51)

After:

![image](https://github.com/nautobot/pynautobot/assets/141754521/62130bd0-24df-4138-b208-3d96d6cc3962)

Reply:
Yes it is. Looks a bit better on non-dark mode @tsm1th

Reply:
This should probably go through dev-standards and the app cookiecutter - we should have consistent styling across all of our docs?

Reply:
I like the idea.

Reply:
Removed this from this PR